pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -115,6 +115,7 @@ Activity:
     - sections
     - lastUpdatedText
     - lastUpdated
+    - canDelete
```

Comment:
@alextanlan @bskinner I noticed the documentation was missing the new `canDelete` property on the full `Activity` object, so I added it.